src/any_agent/config.py

```diff
@@ -53,18 +53,18 @@ class MCPSseParams(BaseModel):
 class TracingConfig(BaseModel):
     model_config = ConfigDict(extra="forbid")
     output_dir: str = "traces"  # None for no json saved trace
-    enable_console: bool = True
-    enable_file: bool = True
+    console: bool = True
+    file: bool = True
+    cost_info: bool = True
```

Comment:
@daavoo  does this rename make it a little less painful to have the bool toggles? cost_info now controls both console printing and general cost tracking.

Reply:
sounds good to me. 
Btw, we should probably add docstrings for each attribute so they are rendered in the API reference

Reply:
one minor thing is I would replace `file` with something more clear, don't know what (json export?)